cocotb/path.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
+
+import os
+import imp
```

Comment:
This is deprecated as of python 3.4 - you should do use `importlib.find_spec('cocotb').origin` there

Reply:
What are you gaining by doing this instead of using `cocotb.__file__`?